### PR TITLE
Case study pages tweaks

### PR DESCRIPTION
**Pull request by izelnakri**
This PR addresses other case study pages styling changes from #1228.

### CODE REVIEW

File: src/ui/styles/blocks/layout.block.css [hunk 1]
```diff
@@ -61,6 +61,7 @@
   .bleed-left,
   .bleed-right {
     display: flex;
+    flex-direction: column;
```
Comment (izelnakri) [inline src/ui/styles/blocks/layout.block.css]: I also cherry-picked this change, do we want this @locks ? Heres the percy diff: https://percy.io/simplabs-com/simplabs.com/builds/6473278?utm_campaign=simplabs-com&utm_content=simplabs.com&utm_source=github_status_private

File: src/ui/components/SidebarQuote/template.hbs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<blockquote>
+  <q>
+    {{yield}}
+  </q>
+  <footer>
+    Peter Steinhäuser, embeDD CEO
```
Comment (marcoow) [inline src/ui/components/SidebarQuote/template.hbs]: This needs to be dynamic as well.

**Review by marcoow — CHANGES_REQUESTED** [review]
I don't think this is quiet right yet – also left some comments on Percy

**Comment by izelnakri** [comment]
after talking with Ricardo, we've decided to keep Ricardo's changes on #1144, and remove them from this PR. Can I have another review @marcoow ? After a squash-merging this, I'll also clean up the qonto PR.

**Review by marcoow — APPROVED** [review]

**Pull request merged**